src/components/share-overlay/share-overlay.js

```diff
@@ -130,6 +131,31 @@ const ShareUrl = (props: Object): React$Element<any> => {
  * @constructor
  */
 const VideoStartOptions = (props: Object): React$Element<any> => {
+  let _inputRefElement = useRef<HTMLInputElement>();
```

Comment:
**non-blocking comment**
best patterns of React(Preact) for inputs are:
1) use uncontrolled inputs, where value takes from ref;
2) use controlled inputs, where value takes from component state;
In current case we have mix with both options, we render input as uncontrolled, but manually change it value inside `onInputChangeHandler` method.
I would recommend add input value inside component state and manage changes there

Reply:
@semarche-kaltura thanks I understand, but i have a question about this- since we are not actually changing the value inside onInputChangeHandler, we just want to prevent invalid chars in this case, do you still think we need to change it? the value that we render is coming from state.startValueFrom and this is the actual meaningful value and we don't want to touch it in case the user inputs an invalid char.